common/apps/apps.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package apps
+
+import (
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+)
+
+type App struct {
+	Image string   // the image reference as supplied by the user on the cli
+	Args  []string // any arguments the user supplied for this app
+	Asc   string   // signature file override for image verification (if fetching occurs)
+
+	// TODO(jonboulle): These images are partially-populated hashes, this should be clarified.
+	ImageID types.Hash // resolved image identifier
+}
+
+type Apps struct {
+	apps []App
+}
+
+// Reset creates a new slice for al.apps, needed by tests
```

Comment:
Methods only used for tests should be declared in test code (it is the same package so should be fine)